content/src/main/content/jcr_root/apps/acs-commons/components/utilities/packager/asset-packager/configuration/dialog.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I know all of the other packagers are done in Classic UI dialogs - do we think it would be a lot of effort for this one to be the first one using a Touch UI dialog?  Eventually all of the other packagers are going to need to convert.

Reply:
I thought about this and my initial desire was to do it in Touch UI, but I actually opted for Classic UI because I think if this packager moves to Touch UI then all the others will have to as well for consistency and I honestly didn't have the time to rework the whole section - it's all structured around the supertype of "acs-commons/components/utilities/packager".
I could raise an issue to have all these converted to Touch UI so the desire is tracked at least if that works.

Reply:
Ok, I think that's fair.